select:MultipleSelect issue

hello,


I try ti use select:MultipleSelect , but got error like 
Caused by: com.google.gwt.core.client.JavaScriptException: (TypeError) : $wnd.jQuery(...).selectpicker is not a function

* the behaviour of select:MultipleSelect  is not consistent , some time is working good

please advice


<b:Row>
	
			<b:Column size="XS_12,SM_12,MD_12,LG_12" offset="XS_0,SM_0,MD_0,LG_0">
			
			<select:MultipleSelect>
			
  				<select:Option text="1"/>
 				<select:Option text="2"/>
 				<select:Option text="3"/>
 				<select:Option text="4"/>
 			
 			</select:MultipleSelect>
			
			
			</b:Column>
		
		</b:Row>



	at Unknown.callback_0_g$(Ctn-0.js@46:33900)
Caused by: com.google.gwt.user.client.ui.AttachDetachException: Exception caught: (TypeError) : $wnd.jQuery(...).selectpicker is not a function
	at Unknown.fillInStackTrace_0_g$(Ctn-0.js@3:30942)
	at Unknown.Throwable_3_g$(Ctn-0.js@8:30897)
	at Unknown.Exception_3_g$(Ctn-0.js@18:31040)
	at Unknown.RuntimeException_3_g$(Ctn-0.js@18:31081)
	at Unknown.UmbrellaException_3_g$(Ctn-0.js@25:57870)
	at Unknown.UmbrellaException_5_g$(Ctn-0.js@26:57931)
	at Unknown.AttachDetachException_1_g$(Ctn-0.js@26:86724)
	at Unknown.tryCommand_1_g$(Ctn-0.js@11:86780)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6051)
	at Unknown.execute_35_g$(Ctn-0.js@10:86808)
	at Unknown.tryCommand_1_g$(Ctn-0.js@14:86764)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6051)
	at Unknown.execute_35_g$(Ctn-0.js@10:86808)
	at Unknown.tryCommand_1_g$(Ctn-0.js@14:86764)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6051)
	at Unknown.execute_35_g$(Ctn-0.js@10:86808)
	at Unknown.tryCommand_1_g$(Ctn-0.js@14:86764)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6051)
	at Unknown.onAttach_1_g$(Ctn-0.js@20:7239)
	at Unknown.setParent_0_g$(Ctn-0.js@12:6147)
	at Unknown.adopt_0_g$(Ctn-0.js@14:6206)
	at Unknown.add_4_g$(Ctn-0.js@8:6264)
	at Unknown.add_3_g$(Ctn-0.js@8:6257)
	at Unknown.add_5_g$(Ctn-0.js@8:6384)
	at Unknown.execute_6_g$(Ctn-0.js@38:11585)
	at Unknown.$executeScheduled_0_g$(Ctn-0.js@40:34159)
	at Unknown.runScheduledTasks_0_g$(Ctn-0.js@9:33881)
	at Unknown.flushPostEventPumpCommands_0_g$(Ctn-0.js@5:33973)
	at Unknown.execute_26_g$(Ctn-0.js@23:34112)
	at Unknown.execute_25_g$(Ctn-0.js@19:33849)
	at Unknown.apply_0_g$(Ctn-0.js@28:33344)
	at Unknown.entry0_0_g$(Ctn-0.js@16:33400)
	at Unknown.anonymous(Ctn-0.js@14:33380)
	at Unknown.callback_0_g$(Ctn-0.js@46:33900)
Caused by: com.google.gwt.core.client.JavaScriptException: (TypeError) : $wnd.jQuery(...).selectpicker is not a function
	at Unknown.jQuery(Unknown@-1)
	at Unknown.initialize_3_g$(Ctn-0.js@23:137166)
	at Unknown.onLoad_12_g$(Ctn-0.js@8:137203)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6052)
	at Unknown.execute_35_g$(Ctn-0.js@10:86808)
	at Unknown.tryCommand_1_g$(Ctn-0.js@14:86764)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6051)
	at Unknown.execute_35_g$(Ctn-0.js@10:86808)
	at Unknown.tryCommand_1_g$(Ctn-0.js@14:86764)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)


my imports 

<!DOCTYPE ui:UiBinder SYSTEM "http://dl.google.com/gwt/DTD/xhtml.ent">
<ui:UiBinder xmlns:ui="urn:ui:com.google.gwt.uibinder"
	xmlns:g="urn:import:com.google.gwt.user.client.ui"	
	xmlns:b="urn:import:org.gwtbootstrap3.client.ui"
    xmlns:b.html="urn:import:org.gwtbootstrap3.client.ui.html"
    xmlns:b.gwt="urn:import:org.gwtbootstrap3.client.ui.gwt"
    xmlns:select="urn:import:org.gwtbootstrap3.extras.select.client.ui"
    xmlns:b3="urn:import:org.gwtbootstrap3.extras.slider.client.ui"
    xmlns:b4="urn:import:org.gwtbootstrap3.extras.toggleswitch.client.ui">

ok ... 
I comment all calls in html file to jquery
<!-- 	<script type="text/javascript" src="jquery-1.10.2.min.js"></script> -->

and now I use jquery from bootstrap core module.

but I got the same error...Any ideas?

Note.The issue appear on on thirst load of view.After firts load its work :-)

errors 

Caused by: com.google.gwt.user.client.ui.AttachDetachException: Exception caught: (TypeError) : $wnd.jQuery(...).selectpicker is not a function
	at Unknown.fillInStackTrace_0_g$(Ctn-0.js@3:30958)
	at Unknown.Throwable_3_g$(Ctn-0.js@8:30913)
	at Unknown.Exception_3_g$(Ctn-0.js@18:31056)
	at Unknown.RuntimeException_3_g$(Ctn-0.js@18:31097)
	at Unknown.UmbrellaException_3_g$(Ctn-0.js@25:57886)
	at Unknown.UmbrellaException_5_g$(Ctn-0.js@26:57947)
	at Unknown.AttachDetachException_1_g$(Ctn-0.js@26:86740)
	at Unknown.tryCommand_1_g$(Ctn-0.js@11:86796)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6051)
	at Unknown.execute_35_g$(Ctn-0.js@10:86824)
	at Unknown.tryCommand_1_g$(Ctn-0.js@14:86780)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6051)
	at Unknown.execute_35_g$(Ctn-0.js@10:86824)
	at Unknown.tryCommand_1_g$(Ctn-0.js@14:86780)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6051)
	at Unknown.execute_35_g$(Ctn-0.js@10:86824)
	at Unknown.tryCommand_1_g$(Ctn-0.js@14:86780)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6051)
	at Unknown.onAttach_1_g$(Ctn-0.js@20:7239)
	at Unknown.setParent_0_g$(Ctn-0.js@12:6147)
	at Unknown.adopt_0_g$(Ctn-0.js@14:6206)
	at Unknown.add_4_g$(Ctn-0.js@8:6264)
	at Unknown.add_3_g$(Ctn-0.js@8:6257)
	at Unknown.add_5_g$(Ctn-0.js@8:6384)
	at Unknown.execute_6_g$(Ctn-0.js@38:11601)
	at Unknown.$executeScheduled_0_g$(Ctn-0.js@40:34175)
	at Unknown.runScheduledTasks_0_g$(Ctn-0.js@9:33897)
	at Unknown.flushPostEventPumpCommands_0_g$(Ctn-0.js@5:33989)
	at Unknown.execute_26_g$(Ctn-0.js@23:34128)
	at Unknown.execute_25_g$(Ctn-0.js@19:33865)
	at Unknown.apply_0_g$(Ctn-0.js@28:33360)
	at Unknown.entry0_0_g$(Ctn-0.js@16:33416)
	at Unknown.anonymous(Ctn-0.js@14:33396)
	at Unknown.callback_0_g$(Ctn-0.js@46:33916)
Caused by: com.google.gwt.core.client.JavaScriptException: (TypeError) : $wnd.jQuery(...).selectpicker is not a function
	at Unknown.jQuery(Unknown@-1)
	at Unknown.initialize_3_g$(Ctn-0.js@23:137182)
	at Unknown.onLoad_12_g$(Ctn-0.js@8:137219)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6052)
	at Unknown.execute_35_g$(Ctn-0.js@10:86824)
	at Unknown.tryCommand_1_g$(Ctn-0.js@14:86780)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)
	at Unknown.onAttach_0_g$(Ctn-0.js@8:6051)
	at Unknown.execute_35_g$(Ctn-0.js@10:86824)
	at Unknown.tryCommand_1_g$(Ctn-0.js@14:86780)
	at Unknown.doAttachChildren_1_g$(Ctn-0.js@3:6219)

--- a/src/main/java/org/gwtbootstrap3/extras/select/client/SelectClientBundle.java
+++ b/src/main/java/org/gwtbootstrap3/extras/select/client/SelectClientBundle.java
@@ -32,8 +32,10 @@ public interface SelectClientBundle extends ClientBundle {
     static final SelectClientBundle INSTANCE = GWT.create(SelectClientBundle.class);
 
     static final String VERSION = "1.10.0";
-    static final String SELECT_JS = "js/bootstrap-select-" + VERSION + ".min.cache.js";
     static final String LOCALE_DIR = "resource/js/locales.cache." + VERSION + "/";
+
+    @Source("resource/js/bootstrap-select-" + VERSION + ".min.cache.js")
+    TextResource select();
 
     @Source(LOCALE_DIR + "defaults-ar_AR.min.js")
     TextResource ar();

--- a/src/main/java/org/gwtbootstrap3/extras/select/client/SelectEntryPoint.java
+++ b/src/main/java/org/gwtbootstrap3/extras/select/client/SelectEntryPoint.java
@@ -21,7 +21,6 @@ package org.gwtbootstrap3.extras.select.client;
  */
 
 import com.google.gwt.core.client.EntryPoint;
-import com.google.gwt.core.client.GWT;
 import com.google.gwt.core.client.ScriptInjector;
 
 /**
@@ -31,7 +30,7 @@ public class SelectEntryPoint implements EntryPoint {
 
     @Override
     public void onModuleLoad() {
-        ScriptInjector.fromUrl(GWT.getModuleBaseURL() + SelectClientBundle.SELECT_JS)
-            .setWindow(ScriptInjector.TOP_WINDOW).setRemoveTag(true).inject();
+        ScriptInjector.fromString(SelectClientBundle.INSTANCE.select().getText())
+            .setWindow(ScriptInjector.TOP_WINDOW).inject();
     }
 }

